ios/Classes/FlutterDocumentView.m

```diff
@@ -113,7 +137,158 @@ - (void)openDocument:(NSString *)document password:(NSString *)password config:(
         fileURL = [NSURL fileURLWithPath:document];
     }
     
+    _documentViewController.delegate = self;
+    flutterResult = result;
```

Comment:
`flutterResult` is an "instance variable". Really this and other "ivars" should where possible be declared as properties. Could you update all of them?

And just for learning, if we were keeping this as an instance variable, it should maintain the naming convention to start with an underscore, so that it's immediately understood by the reader that this is not a local var.